spec/controllers/topics_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe TopicsController, type: :controller do
+  before do
+    login_as build(:user)
+  end
+
+  describe '#update' do
+    context 'when publishing' do
+      it 'publishes the topic to the publishing api' do
+        stub_any_publishing_api_publish
+        stub_any_rummager_post
+        topic = create(:topic)
+
+        put :update, id: topic.id, publish: true
+
+        assert_publishing_api_publish(topic.content_id)
+      end
+
+      it 'sends the topic to rummager' do
+        stub_any_publishing_api_publish
+        stub_any_rummager_post
+        topic = create(:topic, path: '/service-manual/a-topic')
+
+        put :update, id: topic.id, publish: true
+
+        assert_rummager_posted_item(_id: '/service-manual/a-topic')
+      end
+    end
+  end
+end
```

Comment:
Erm.

Honestly, I have no idea. It probably seemed like a good idea at the time, but I wrote this a month ago. These can probably be removed, if they're not adding anything.